compiler/one-cmds/one-import-onnx

```diff
@@ -322,61 +320,81 @@ def _convert(args):
-            alt_path = os.path.join(tmpdir, os.path.splitext(basename)[0] + '-alt.onnx')
-            onnx.save(onnx_model, alt_path)
-
-            # call extension with options
-            ext_cmd = [ext_path]
-            if oneutils.is_valid_attr(args, 'unroll_rnn'):
-                ext_cmd.append('--unroll_rnn')
-            if oneutils.is_valid_attr(args, 'unroll_lstm'):
-                ext_cmd.append('--unroll_lstm')
-            if oneutils.is_valid_attr(args, 'experimental_disable_batchmatmul_unfold'):
-                ext_cmd.append('--experimental_disable_batchmatmul_unfold')
-            if oneutils.is_valid_attr(args, 'save_intermediate'):
-                ext_cmd.append('--save_intermediate')
-            if oneutils.is_valid_attr(args, 'keep_io_order'):
-                ext_cmd.append('--keep_io_order')
-            ext_cmd.append(alt_path)
-            ext_cmd.append(getattr(args, 'output_path'))
-            oneutils.run(ext_cmd, logfile=f)
-            return
-
-        tf_savedmodel = onnx_tf.backend.prepare(onnx_model)
-
-        savedmodel_name = os.path.splitext(os.path.basename(
-            args.output_path))[0] + '.savedmodel'
-        savedmodel_output_path = os.path.join(tmpdir, savedmodel_name)
-        tf_savedmodel.export_graph(savedmodel_output_path)
-
-        # make a command to convert from tf to tflite
-        tf2tfliteV2_path = os.path.join(dir_path, 'tf2tfliteV2.py')
-        tf2tfliteV2_output_name = os.path.splitext(os.path.basename(
-            args.output_path))[0] + '.tflite'
-        tf2tfliteV2_output_path = os.path.join(tmpdir, tf2tfliteV2_output_name)
-
-        tf2tfliteV2_cmd = _make_cmd.make_tf2tfliteV2_cmd(args, tf2tfliteV2_path,
-                                                         savedmodel_output_path,
-                                                         tf2tfliteV2_output_path)
-
-        f.write((' '.join(tf2tfliteV2_cmd) + '\n').encode())
-
-        # convert tf to tflite
-        oneutils.run(tf2tfliteV2_cmd, logfile=f)
-
-        # make a command to convert from tflite to circle
-        tflite2circle_path = os.path.join(dir_path, 'tflite2circle')
-        tflite2circle_cmd = _make_cmd.make_tflite2circle_cmd(tflite2circle_path,
-                                                             tf2tfliteV2_output_path,
-                                                             getattr(args, 'output_path'))
-
-        f.write((' '.join(tflite2circle_cmd) + '\n').encode())
-
-        # convert tflite to circle
-        oneutils.run(tflite2circle_cmd, err_prefix="tflite2circle", logfile=f)
+            ext_alt_onnx_path = os.path.join(tmpdir,
+                                             os.path.splitext(basename)[0] + '-alt.onnx')
+            onnx.save(onnx_model, ext_alt_onnx_path)
+
+            if _force_ext(args):
+                run_default_import = False
+
+        res_conv = -1
+        if run_default_import:
```

Comment:
Even if `force_ext=True`, onnx-tf is used if `onnx-import-onnx-ext` is not installed. Is this intended? How about throwing an exception for that case (ex: "force_ext is True, but onnx-import-onnx-ext is not installed.")?

Reply:
Or, if you're going to keep the current behavior, it would be helpful to print logs, ex: "force_ext is True, but onnx-import-onnx-ext is not installed. onnx-tf is used."

Why? I think it'll be helpful to debug our customers' problems with their logs.

Reply:
> Why? I think it'll be helpful to debug our customers' problems with their logs.

Good point!